### PR TITLE
KRACOEUS-8826: fixing DD file descriptions

### PR DESCRIPTION
**Pull request by leoherbie**

### CODE REVIEW

File: coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml [hunk 1]
```diff
@@ -399,11 +399,11 @@
...
-    <property name="label" value="Award ID" />
-    <property name="shortLabel" value="Award ID" />
+    <property name="label" value="Award Number" />
+    <property name="shortLabel" value="Award Number" />
     <property name="required" value="false" />
-    <property name="summary" value="Award ID" />
-    <property name="description" value="Award ID" />
+    <property name="summary" value="Award Number" />
+    <property name="description" value="Award Number" />
   </bean>
```
Comment (tdurkin) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: How long have we had the Award Number and ID's backwards? I think we need to have the Award Subcommittee co-chairs weigh in on changing this if it's been this way a long time.

Comment (leoherbie) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: I'll check

Comment (leoherbie) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: b8d977e	11/13/09, 12:51 PM	Jack Frosch	KCAWD-425: Fixed Award Lookup labels to show Award ID for award number field. Also patched a referential integrity hole for Award unit number.

Comment (tdurkin) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: Let's hold off on this change until we get more discussion/consensus.  I know it's weird and backwards now, but changing that in 6.0.1 will be problematic.  Can you revert that part and I'll merge the other changes?

Comment (leoherbie) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: sure

Comment (leoherbie) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: What about Sponsor Award Number being called Sponsor Award ID?

Comment (tdurkin) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: Yeah - Revert that too, please.

Comment (leoherbie) [inline coeus-impl/src/main/resources/org/kuali/kra/datadictionary/Award.xml]: ok

**Comment by leoherbie** [comment]
reverted.  please merge

**Pull request merged**